products/vpcaccess/api.yaml

```diff
@@ -84,12 +84,12 @@ objects:
         output: true
         exclude: true
         values:
-          - STATE_UNSPECIFIED
-          - READY
-          - CREATING
-          - DELETING
-          - ERROR
-          - UPDATING
+          - :STATE_UNSPECIFIED
```

Comment:
This change looks good, but not relevant to this resource. Can it be made separately?